tutorials/parser/src/main/java/com/datatorrent/tutorial/csvparser/csvParserApplication.java

```diff
@@ -0,0 +1,28 @@
+package com.datatorrent.tutorial.csvparser;
+
+import org.apache.hadoop.conf.Configuration;
+
+import com.datatorrent.api.DAG;
+import com.datatorrent.api.StreamingApplication;
+import com.datatorrent.api.annotation.ApplicationAnnotation;
+import com.datatorrent.contrib.parser.CsvParser;
+import com.datatorrent.lib.io.ConsoleOutputOperator;
+
+@ApplicationAnnotation(name = "csvParseApplication")
+public class csvParserApplication implements StreamingApplication
+{
+
+  @Override
+  public void populateDAG(DAG dag, Configuration conf)
+  {
+    AddDataGenerator dataGenerator = dag.addOperator("dataGenerator", new AddDataGenerator());
+    CsvParser parserOperator = dag.addOperator("csvParser", new CsvParser());
+    ConsoleOutputOperator consoleOperator = dag.addOperator("dataconsole", new ConsoleOutputOperator());
+    ConsoleOutputOperator errorConsoleOperator = dag.addOperator("errorConsole", new ConsoleOutputOperator());
+
+    dag.addStream("inputData", dataGenerator.out, parserOperator.in);
+    dag.addStream("parsedData", parserOperator.parsedOutput, consoleOperator.input);
+    dag.addStream("errorData", parserOperator.err, errorConsoleOperator.input);
```

Comment:
I would suggest we add one more console operator that would receive POJOs. Most downstream operators expect a POJO hence having it in example would be good.